lib/uploader.ts

```diff
@@ -508,6 +515,11 @@ export class Uploader {
 					await Promise.all(chunksQueue)
 					this.updateStats()
 
+					// re-add upload because it was reset
+					this._uploadQueue.push(upload)
+					upload.status = UploadStatus.ASSEMBLING
```

Comment:
this part looks really ugly but I don't know hot to make it better. This happens because of:

```js
// Reset when upload queue is done
this._jobQueue.addListener('idle', () => this.reset())
```

so, when chunks are uploaded - we're detaching `upload` from the queue and status change for this `upload` not reflected in vue components

Reply:
Fixed in https://github.com/nextcloud-libraries/nextcloud-upload/pull/1521